### PR TITLE
Feedback on 2010 Pair Project

### PR DESCRIPTION
**Pull request by scottalexandra**

### CODE REVIEW

File: module2/projects/pair_project_2010.md [hunk 1]
```diff
@@ -88,9 +88,9 @@ title: BE Mod 2 Week 1-2 Pair Project
...
-| **2: Passing with Concerns** | Students complete all but 3-5 User Stories | Students utilize MVC to organize code, but cannot defend some of their design decisions. | Ruby is used to process data that could use ActiveRecord instead. | Feature test coverage between 90% and 98%, or model test coverage below 100%, or tests are not meaningfully written or have an unclear objective. |
+| **4: Exceptional**  | All User Stories 100% complete including edge cases, and at least one extension story completed | Students use the principles of MVC to effectively organize code. | Highly effective and efficient use of ActiveRecord beyond what we've taught in class. Even `.each` calls will not cause additional database lookups (We load the data as efficiently as possible every time we visit the DB). | Very clear Test Driven Development. Test files are extremely well organized and nested. Students utilize `before :each` blocks. 100% coverage for features and models, sad path testing, and a 4 on Feature Completeness |
```
Comment (scottalexandra) [inline module2/projects/pair_project_2010.md:91]: At this point in the mod do we care that much about students utilizing MVC? Is the emphasis more on get the functionality to work? If so, I moved utilizing MVC to a 4 and left 3 and below to be more about defending design decisions and finding a way to organize code even if it's not MVC.

**Review by timomitchel — APPROVED** [review]
Thank you!
